### ISSUE TITLE
Yubikey support

### ISSUE THREAD

**Issue by candroid-man** [streetwriters/notesnook#428]
**Feature Request**

Once 2FA rolls out, YubiKey support would be really nice to see. Not many apps have it and people in the privacy community love them

**Comment by thecodrr** [comment]
I agree. I have already researched on this and it won't take much time. The only issue is acquiring a security key for testing.

**Comment by neorrk** [comment]
I hope you can implement this. There are also ways to use a mobile phone as a FIDO2 hardware key.

**Comment by MFYDev** [comment]
@thecodrr Also want this feature, as this feature can hugely enhance the security of the notesnook

**Comment by nguyenhoang158** [comment]
A highly anticipated feature

**Comment by gh-bravo-sierra** [comment]
Any updates here? Do we need to raise as a feature request? 🔒

**Comment by butterdawgs** [comment]
Still waiting for this one.
